spring-vault-core/src/main/java/org/springframework/vault/support/CertificateBundle.java

```diff
@@ -87,18 +84,19 @@ public class CertificateBundle extends Certificate {
 	 * @param certificate must not be empty or {@literal null}.
 	 * @param issuingCaCertificate must not be empty or {@literal null}.
 	 * @param privateKey must not be empty or {@literal null}.
+	 * @param revocationTime the revocation time.
 	 * @return the {@link CertificateBundle} instead.
 	 */
 	public static CertificateBundle of(String serialNumber, String certificate, String issuingCaCertificate,
-			String privateKey) {
+			String privateKey, long revocationTime) {
```

Comment:
We should introduce a new static factory method instead of updating the existing one.

Reply:
Updated the code